internal/testserver/datastore/postgres.go

```diff
@@ -32,7 +32,7 @@ func RunPostgresForTesting(t testing.TB, bridgeNetworkName string) RunningEngine
 	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
 		Name:         name,
 		Repository:   "postgres",
-		Tag:          "10.20",
+		Tag:          "13.8",
```

Comment:
I think it might be worth moving this version into a const in the postgres datastore itself

Reply:
I think the min required version (13.0) and the test version (latest 13.x) aren't necessarily the same? I can see postgres advertising its min required version.